src/main/resources/assets/inventorytweaks/lang/en_US.lang

```diff
@@ -39,6 +39,8 @@ invtweaks.settings.rulesfile=Open the sorting rules file...
 invtweaks.settings.rulesfile.error=[8] Failed to open rules file
 invtweaks.settings.treefile=Open the item tree file...
 invtweaks.settings.treefile.error=[9] Failed to open tree file
+invtweaks.settings.moddedtreefile=Load modded tree file
+invtweaks.settings.moddedtreefile.error=Failed to replace tree file
```

Comment:
This should probably be prefixed with a number: [14] looks like the first unused one (and wouldn't require all the translations to be updated) even though that breaks the pattern of going from top to bottom in the file